src/test/java/bio/terra/service/rawls/RawlsServiceTest.java

```diff
@@ -87,77 +94,105 @@ public void setUp() throws Exception {
     // Mock out the lower level client in individual as needed
     when(samResourceApi.getApiClient()).thenAnswer(a -> apiClient);
 
-    workspaceId = UUID.randomUUID();
-    workspaceName = "testWorkspace";
-    workspaceNamepace = "testNamespace";
-
-    when(rawlsClient.getWorkspace(any(), any()))
```

Comment:
With this broad mocking, we didn't register that we never called `rawlsClient.getWorkspace` for our `workspaceId`.